include/c2pa.hpp

```diff
@@ -202,6 +202,11 @@ namespace c2pa
         /// @return The number of bytes written.
         /// @throws C2pa::C2paException for errors encountered by the C2PA library.
         int64_t get_resource(const string &uri, std::ostream &stream);
+
+        /// @brief Get the raw C2paReader pointer.
+        /// @return The raw C2paReader pointer.
+        /// @note This is intended for internal use and compatibility with C APIs.
+        C2paReader* get_raw_reader() const { return c2pa_reader; }
```

Comment:
Your call here, but I wonder if you should name this something like `api_internal_get_raw_pointer?`. I can imagine AI or autocomplete pulling this up without the developer reading the docs. Moreover, we could use the [[deprecated]] attribute to further dissuade folks from using this. Up to you, though!

Reply:
Won't using [[deprecated]] create warnings for us, that, depending on settings, would fail our builds too?

Reply:
For sure--we could disable for this case explicitly. Hopefully, marking it deprecated would prevent other folks from using it because of their compiler settings prevent the use of deprecated functions. 

I went back and forth about whether to include the [[deprecated]] suggestion. NBD if you'd prefer to not include the deprecated attribute 👍

Reply:
I won't add it for now - but I renamed the function as you suggested.